packages/cadl-ranch-specs/http/payload/xml/main.tsp

```diff
@@ -32,6 +34,42 @@ model ModelWithOptionalField {
   value?: int32;
 }
 
+@doc("Contains fields that are XML attributes.")
+model ModelWithAttributes {
+  @attribute id1: int32;
+  @attribute id2: string;
+  @name("Enabled") enabled: boolean;
```

Comment:
could we keep trhe renaming of props as a separate dedicated test.

Should also add one that is done with `@encodedName("application/xml", "abc")` which should be equivalent to Xml.name

Reply:
done